src/components/storage/Common.jsx

```diff
@@ -72,22 +73,20 @@ export const useDuplicateDeviceNames = ({ deviceNames }) => {
     return duplicateDeviceNames;
 };
 
-export const useHasFilesystems = ({ selectedDisks, devices }) => {
-    const [hasFilesystems, setHasFilesystems] = useState();
+export const useUsablePartitions = ({ selectedDisks, devices }) => {
+    const [usablePartitions, setUsablePartitions] = useState();
 
     useEffect(() => {
-        const _hasFilesystems = (
-            selectedDisks.some(device => (
-                devices[device]?.children.v.some(child => (
-                    devices[child]?.formatData.mountable.v || devices[child]?.formatData.type.v === "luks")
-                )
-            ))
-        );
-
-        setHasFilesystems(_hasFilesystems);
+        const _usablePartitions = Object.values(devices).filter(device =>
+            (device.formatData?.type.v === "biosboot" ||
```

Comment:
Oh, I have just noticed, having the string here does not seem right.
It should be more generic check or it should at least include also "prepboot" and "appleboot".
Something for a potential follow-up I think.